week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -57,8 +76,49 @@ function exerciseTwo(shopping) {
 
     The end result should look something like this: https://hyf-js2-week1-makeme-ex1-demo.herokuapp.com/
 **/
+// const books = [
+//   {
+//     title: “The Design of Everyday Things”,
+//   author: “Don Norman”,
+//   alreadyRead: false
+//   },
+// {
+//   title: “The Most Human Human”,
+//   author: “Brian Christian”,
+//   alreadyRead: true
+// },
+// {
+//   title: “The Pragmatic Programmer”,
+//   author: “Andrew Hunt”,
+//   alreadyRead: true
+// }
+// ];
+let url1 =
+  "https://images-na.ssl-images-amazon.com/images/I/410RTQezHYL._SX326_BO1,204,203,200_.jpg";
+let url2 =
+  "https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1295465264l/8884400.jpg";
+let url3 =
+  "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRvVg-9fpJlcF2GWF2Vd5c181LSCsT9VwZkT844SmzohNLtyRGcwIKvn43eLLx4biDzCfg_YwDi&usqp=CAc";
+let urls = [url1, url2, url3];
 function exerciseThree(books) {
-  //Write your code in here
+  let list = document.createElement("ul");
+  let content = document.querySelector("#content");
+  content.appendChild(list);
+  let urls = [url1, url2, url3];
```

Comment:
You have already created this array on line 102. How about you extend the books array object with a new `url` property rather than handling this with a new array?
eg  
```
 {
    title: "The Design of Everyday Things",
    author: "Don Norman",
    alreadyRead: false,
    url: "https://images-na.ssl-images-amazon.com/images/I/410RTQezHYL._SX326_BO1,204,203,200_.jpg"
  }
```